subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/inception/GradleInceptionPerformanceTest.groovy

```diff
@@ -44,7 +44,7 @@ import static org.gradle.test.fixtures.server.http.MavenHttpPluginRepository.PLU
  */
 @Issue('https://github.com/gradle/gradle-private/issues/1313')
 @Category(SlowPerformanceRegressionTest)
-class GradleInceptionPerformanceTest extends AbstractCrossVersionGradleInternalPerformanceTest {
+class GradleInceptionPerformanceTest extends AbstractCrossVersionGradleProfilerPerformanceTest {
```

Comment:
Can we rename this class? I doesn't test the `gradle/gradle` build at all.

Reply:
Done.